sale_financial_risk/models/res_partner.py

```diff
@@ -39,7 +39,6 @@ def _compute_risk_sale_order(self):
             domain=self._get_risk_sale_order_domain(),
             fields=["risk_partner_id", "company_id", "risk_amount"],
             groupby=["risk_partner_id", "company_id"],
-            orderby="id",
             lazy=False,
```

Comment:
@pedrobaeza, I think that the order by must belong to the fields list. The id don't appear in result :
example : 
'''
>>> lines = self.env['sale.order.line'].read_group(domain, fields=['product_id'], groupby=['product_id'], orderby='id desc')
>>> lines
[{'product_id_count': 2, 'product_id': (21, <odoo.tools.func.lazy object at 0x7fdf6494a438>), '__domain': [('product_id', '=', 21)]}, {'product_id_count': 11, 'product_id': (25, <odoo.tools.func.lazy object at 0x7fdf6494a480>), '__domain': [('product_id', '=', 25)]}, {'product_id_count': 2, 'product_id': (32, <odoo.tools.func.lazy object at 0x7fdf64945f30>), '__domain': [('product_id', '=', 32)]}, {'product_id_count': 1, 'product_id': (35, <odoo.tools.func.lazy object at 0x7fdf64945090>), '__domain': [('product_id', '=', 35)]}, {'product_id_count': 1, 'product_id': (12, <odoo.tools.func.lazy object at 0x7fdf6494a3f0>), '__domain': [('product_id', '=', 12)]}, {'product_id_count': 1, 'product_id': (34, <odoo.tools.func.lazy object at 0x7fdf64945120>), '__domain': [('product_id', '=', 34)]}, {'product_id_count': 2, 'product_id': (4, <odoo.tools.func.lazy object at 0x7fdf64945990>), '__domain': [('product_id', '=', 4)]}, {'product_id_count': 4, 'product_id': (3, <odoo.tools.func.lazy object at 0x7fdf6494a318>), '__domain': [('product_id', '=', 3)]}, {'product_id_count': 5, 'product_id': (5, <odoo.tools.func.lazy object at 0x7fdf6494a360>), '__domain': [('product_id', '=', 5)]}, {'product_id_count': 5, 'product_id': (6, <odoo.tools.func.lazy object at 0x7fdf6494a3a8>), '__domain': [('product_id', '=', 6)]}, {'product_id_count': 15, 'product_id': (31, <odoo.tools.func.lazy object at 0x7fdf6494a4c8>), '__domain': [('product_id', '=', 31)]}]
'''

Reply:
And this is a performance thing for avoiding an unneeded extra sorting, which is critical for getting good results. I don't understand why you insist in this without giving reasons.